### PR TITLE
Readme

### LINKED ISSUE #10: Complete ReadMe for BE/API

### PR DESCRIPTION
**Pull request by CaliHam**
What I did:
Add info for readme. This PR will close #10 

Did this break anything?

- [x] No
- [ ]  Yes

Type of change

- [x] New feature (non-breaking change which adds functionality)
- [ ]  Bug fix (non-breaking change which fixes an issue)
- [ ]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ]  Super small fix (Corrected a typo, removed a comment, etc.)
- [ ]  Skip all the other stuff and briefly explain the fix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
-# bare-minimum-api
+# Bare-Minimum API
+### [View The Deployed API Here](https://bare-minimum-api-53c62eb03bf8.herokuapp.com)
```
Comment (taranveersingh93) [inline README.md:2]: Could we say "access" instead of "view" as the backend repo is essentially a server so it's not "visible"

File: README.md [hunk 2]
```diff
@@ -1 +1,66 @@
...
+|       `/api/v1/savedtasks`        |      post to saved tasks      |    POST     |               none               |
+|       `/api/v1/savedtasks:id`        |      patch saved task      |    PATCH     |               none               |
+|       `/api/v1/savedtasks:id`        |      delete saved task      |    DELETE     |               none               |
+
+## Future Extensions
+ - [ ] Add search feature within saved tasks
+ - [ ] Add a suggestions form for the user to suggest a task for a category
+ 
+## Tools & Technologies
+ - SQL
```
Comment (taranveersingh93) [inline README.md:42]: Should we list SQL and postgresQl separately?

**Review by taranveersingh93 — CHANGES_REQUESTED** [review]

**Review by SethBurns — APPROVED** [review]
Approved pending Taranveer's change requests.

**Review by taranveersingh93 — APPROVED** [review]

**Review by hvohr — APPROVED** [review]
Love this! thank you!

**Pull request merged**